test/DotNetCore.CAP.Test/Nats/NatsTransportTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
There is no need to add integration tests for each Transport, so it can be removed